qctrlopencontrols/dynamic_decoupling_sequences/predefined.py

```diff
@@ -638,31 +723,69 @@ def new_periodic_sequence(
 def new_walsh_sequence(
     duration=None, paley_order=None, pre_post_rotation=False, **kwargs
 ):
-    """
-    Creates Walsh Single Axis Sequence.
+    r"""
+    Creates the Walsh sequence.
 
     Parameters
     ---------
     duration : float
-        Total duration of the sequence. Defaults to None
+        Total duration of the sequence :math:`\tau`. Defaults to None.
     paley_order : int, optional
-        Defaults to 1. The paley order of the walsh sequence.
+        The paley order :math:`k` of the walsh sequence. Defaults to 1.
```

Comment:
```suggestion
        The paley order :math:`k` of the Walsh sequence. Defaults to 1.
```

I think Walsh is capitalized in the rest of the documentation